### PR TITLE
regenerated test vector to add a valid DID for to field in `Message.metadata.from|to`

### PR DESCRIPTION
**Pull request by jiyoontbd**

### CODE REVIEW

File: hosted/test-vectors/protocol/vectors/parse-close.json
Comment (jiyoontbd) [inline hosted/test-vectors/protocol/vectors/parse-close.json:3]: all these were regenerated because Message.metadata.to field was `did:ex:alice` and that fails our schema validation - it must be a real DID

**Comment by KendallWeihe** [comment]
Could you add more description as to what made the pre-existing DIDs invalid? (or maybe point to a ticket which describes the issue)

I'm cool with the CODEOWNERs change, though it's worthy of a discussion (changes here being so upstream of everything else makes changes here have leverage, resulting in large downstream implications; my point being, if there ever were a need to create a constitution of governance then it should be had here -- "we're not there yet" is a valid response such that we can iterate quicker right now). Which, there may have been a discussion for which I wasn't privy to, but if not, could we make the CODEOWNERs a dedicated PR and/or ticket on the backlog?

**Comment by jiyoontbd** [comment]
> Could you add more description as to what made the pre-existing DIDs invalid? (or maybe point to a ticket which describes the issue)

yes, can do! so i found this while getting my tbdex-js pr up to speed with main, and found that tbdex had some schema changes that enforced `Message.metadata.from/to` to be a real DID. the test vector was not updated with this change, and some of the test vectors had the value for those fields set to an invalid DID like `did:ex:alice` or `did:ex:pfi`. tests in tbdex-js and tbdex-kt that used the test vectors were failing because of this

> I'm cool with the CODEOWNERs change, though it's worthy of a discussion (changes here being so upstream of everything else makes changes here have leverage, resulting in large downstream implications; my point being, if there ever were a need to create a constitution of governance then it should be had here -- "we're not there yet" is a valid response such that we can iterate quicker right now). Which, there may have been a discussion for which I wasn't privy to, but if not, could we make the CODEOWNERs a dedicated PR and/or ticket on the backlog?

happy to move the codeowners change out to a separate PR!

File: hosted/json-schemas/message.schema.json
Comment (jiyoontbd) [inline hosted/json-schemas/message.schema.json:14]: changed to be consistent with how we use $ref elsewhere https://github.com/TBD54566975/tbdex/pull/247/files#diff-781d77c475148a35130d81220b3a6d100357fc53576806be6dda9a528f1ae283L10

**Comment by KendallWeihe** [comment]
> the test vectors had the value for those fields set to an invalid DID like `did:ex:alice` or `did:ex:pfi`

Got it, thanks! A good amount of these seem like `did:dht`-to-`did:jwk` changes. I would imagine we want `did:dht` in some of our test vectors, albeit maybe not currently? Or maybe it's beside the point and either or are fine and the changes here were just a byproduct of you generating new test vectors. Just raising awareness cc @phoebe-lew

**Comment by jiyoontbd** [comment]
> > the test vectors had the value for those fields set to an invalid DID like `did:ex:alice` or `did:ex:pfi`
> 
> Got it, thanks! A good amount of these seem like `did:dht`-to-`did:jwk` changes. I would imagine we want `did:dht` in some of our test vectors, albeit maybe not currently? Or maybe it's beside the point and either or are fine and the changes here were just a byproduct of you generating new test vectors. Just raising awareness cc @phoebe-lew

yes, so i actually did initially opt to use did:dht, but found an issue while using did:dht:... for DIDs in test vectors and then ran tests in tbdex-kt and ran into an issue. https://github.com/TBD54566975/tbdex-kt/issues/180

i should probably write an issue to revert the DIDs back to did:dht once that issue is resolved. wrote issue here: https://github.com/TBD54566975/tbdex/issues/249

**Review by diehuxx — APPROVED** [review]

**Comment by jiyoontbd** [comment]
So would that mean we have to first address the did:dht issue i found while
running test vector tests in tbdex-kt? I only used did:jwk to unblock
myself so all the tests would pass with the latest test vectors.

**Comment by mistermoe** [comment]
@jiyoontbd just fixed the issue you found earlier! 

Related issue:  https://github.com/TBD54566975/tbdex-kt/issues/180
PR: https://github.com/TBD54566975/web5-kt/pull/251

**Review by diehuxx — APPROVED** [review]

**Review by KendallWeihe — APPROVED** [review]

**Pull request merged**
